build/ci/macos/dumpsyms.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+echo "Generate dump symbols"
+
+GEN_SCRIPT=tools/crashdump/generate_syms.sh
+DUMPSYMS_BIN=$HOME/breakpad/dump_syms
+ARTIFACTS_DIR=build.artifacts
+BUILD_DIR=build.release
+SYMBOLS_DIR=$ARTIFACTS_DIR/symbols
+MSCORE_BIN=$BUILD_DIR/main/mscore.app/Contents/MacOS/mscore
```

Comment:
Shouldn't that be `MSCORE_BIN=$BUILD_DIR/main/mscore.app/Contents/MacOS/bin/mscore`?

Reply:
Instead of all those `echo`s you could use `set -x` and see every command and every variable being printed

Reply:
The fact is that I need to find out not where the error is (where the error I know is not the right path). I need to find the correct path to the compiled binary file (locally on my machine, I have it different, not the same as on the server).

I changed the path to the path to the binary file in the installation directory (and not in the build directory, as I originally wanted), this will also work.

Reply:
I know that: 
* build.release/main/mscore.app/Contents/MacOS/mscore - path in the build dir to mscore binary on my local machine   
* applebuild/mscore.app/Contents/MacOS/mscore - path in the install dir to mscore binary on CI   

But I do not know what is the path to the ьscore binary in the build dir on CI.
It looks like this -  build.release/main/mscore.build/Release/mscore.app/Contents/MacOS/mscore 
But until the end I did not find out, I decided to specify the path from the install dir, this will also work.

Reply:
I understood your advice about set -x.  Thanks :)